src/artwork-preview.js

```diff
@@ -19,19 +19,29 @@ var ArtworkPreview = React.createClass({
     var title = showLink ? 
       <Link to="artwork" params={{id: art.id}}>{art.title}</Link> :
       art.title
+      
+    var details = showLink ? <div className="objects-page-link"><div className="objects-page-icon"></div>details</div> : ''
```

Comment:
This button doesn't do anything because it's a div with another div inside of it. It needs to link to the object page. I fixed it in the version pushed to staging